### PR TITLE
vscode-extensions.github.copilot-chat: 0.13.2024022301 -> 0.14.2024032901

### PR DESCRIPTION
**Pull request by Tobi34**
## Description of changes
update github.copilot-chat from 0.13.2024022301 to 0.14.2024032901

## Things done

I have checked  that the installed packets work with vscode 1.88.1. (with extensionsFromVscodeMarketplace )

### CODE REVIEW

File: pkgs/applications/editors/vscode/extensions/default.nix
Comment (Laurent2916) [inline pkgs/applications/editors/vscode/extensions/default.nix:1776]:
```suggestion
          version = "0.14.2024032901";  # compatible with vscode 1.88.1
```

**Review by Laurent2916 — CHANGES_REQUESTED** [review]
typo, also the commit message should be:
"vscode-extensions.github.copilot-chat: 0.13.2024022301 -> 0.14.2024032901"

**Comment by Laurent2916** [comment]
Result of `nixpkgs-review pr 305830` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions.github.copilot-chat</li>
  </ul>
</details>

**Review by Laurent2916 — APPROVED** [review]

**Comment by Laurent2916** [comment]
> How will these continue ?

Well now that it's approved, we simply need to wait for someone with merge permissions to merge your PR!

**Review by superherointj — APPROVED** [review]

**Pull request merged**
